test/workflow-orchestration.ts

```diff
@@ -435,4 +439,65 @@ describe('Workflow chaining for a collection configured for swot reprojection an
       expect(workflowSteps[1].serviceID).to.equal('sds/swot-reproject:latest');
     });
   });
+});
+
+describe('Request that spans multiple CMR pages', function () {
```

Comment:
It looks like this test must be somehow limiting the default query cmr page size to 3. Where is that happening?